editor/script_create_dialog.cpp

```diff
@@ -44,6 +44,16 @@ void ScriptCreateDialog::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_THEME_CHANGED:
 		case NOTIFICATION_ENTER_TREE: {
+			for (int i = 0; i < ScriptServer::get_language_count(); i++) {
+				String lang = ScriptServer::get_language(i)->get_name();
+				language_menu->set_item_icon(i, get_icon(lang, "EditorIcons"));
+			}
+			String last_lang = EditorSettings::get_singleton()->get_project_metadata("script_setup", "last_selected_language", "");
+			if (!last_lang.empty()) {
+				language_menu->set_icon(get_icon(last_lang, "EditorIcons"));
```

Comment:
Setting `OptionButton` icon should be handled by the `select` method, but it doesn't work if done from code, not sure if that's a bug. When selected manually, the `OptionButton` icon is set properly.